fdbclient/include/fdbclient/GetEncryptCipherKeys.actor.h

```diff
@@ -25,14 +25,51 @@
 #define FDBCLIENT_GETCIPHERKEYS_ACTOR_H
 
 #include "fdbclient/EncryptKeyProxyInterface.h"
+#include "fdbrpc/Stats.h"
 #include "flow/BlobCipher.h"
+#include "flow/Knobs.h"
 #include "flow/IRandom.h"
 
 #include <unordered_map>
 #include <unordered_set>
 
 #include "flow/actorcompiler.h" // This must be the last #include.
 
+struct BlobCipherKeyCacheMetrics {
```

Comment:
nit: move the struct definition to BlobCipher.h?

Reply:
No we cannot before we move BlobCipher to fdbclient. We need to include fdbrpc/Stats.h